src/store/modules/channel/actions.ts

```diff
@@ -70,6 +71,140 @@ const actions: ActionTree<ChannelState, RootState> = {
     commit(types.CHANNEL_INVENTORY_CHANNELS_UPDATED, groups);
   },
 
+  async fetchShopifyConfigs ({ commit }) {
+    let shopifyConfigs = [];
+
+    try {
+      const payload = {
+        inputFields: {
+          productStoreId: store.state.user.currentEComStore.productStoreId,
+        },
+        fieldList: ["shopifyConfigId", "name", "shopId"],
+        entityName: "ShopifyShopAndConfig",
+        noConditionFind: "Y"
+      }
+
+      const resp = await ChannelService.fetchShopifyConfigs(payload);
+      if (!hasError(resp)) {
+        shopifyConfigs = resp.data.docs;
+      } else {
+        throw resp.data
+      }
+    } catch(error: any) {
+      logger.error(error)
+    }
+    return shopifyConfigs;
+  },
+
+  async fetchJobs ({ commit, dispatch }) {
```

Comment:
Fetch jobs and configs again on changing productStore

Reply:
Its already handled in the code.